azure-pipelines-official.yml

```diff
@@ -103,7 +103,6 @@ stages:
                 -officialSourceBranchName $(SourceBranchName)
                 -officialIbcSourceBranchName $(IbcSourceBranchName)
                 -officialIbcDropId $(IbcDropId)
-                -skipAnalyzers
```

Comment:
Analyzer execution has been switched from _opt-out_ mode to _opt-in_ mode in build scripts.

`skipAnalyzers` flag has been replaced with `runAnalyzers` flag, which defaults to `false`. It is only explicitly set to `true` for `build-correctness` test leg.

Reply:
Should the `runAnalyzers` flag be documented here?